bundles/org.openhab.binding.millheat/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,146 @@
+		<item-type>Number:Temperature</item-type>
+		<label>Target Temperature</label>
+		<category>Heating</category>
+		<tags>
+			<tag>TargetTemperature</tag>
+		</tags>
+		<state pattern="%d %unit%" min="5" max="35" step="1" />
+	</channel-type>
+	<channel-type id="targetTemperatureRoom">
+		<item-type>Number:Temperature</item-type>
+		<label>Target Temperature</label>
+		<category>Heating</category>
+		<tags>
+			<tag>TargetTemperature</tag>
+		</tags>
+		<state pattern="%d %unit%" readOnly="true" min="5" max="35" step="1" />
+	</channel-type>
+	<channel-type id="heatingActive">
+		<item-type>Switch</item-type>
+		<label>Heating Active</label>
+		<description>Current state of the heater or heaters in room</description>
+		<category>Energy</category>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="independent">
+		<item-type>Switch</item-type>
+		<label>Independent Heater</label>
+		<description>ON if heater is an independent heater and not connected to a room</description>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="masterSwitch">
+		<item-type>Switch</item-type>
+		<label>Master Switch</label>
+		<description>Master ON/OFF switch for independent heater</description>
+	</channel-type>
+	<channel-type id="fanActive">
+		<item-type>Switch</item-type>
+		<label>Fan Active</label>
+		<description>Current state of heater fan (if available, OFF if not found)</description>
+		<category>Flow</category>
+	</channel-type>
+	<channel-type id="window">
+		<item-type>Contact</item-type>
+		<label>Window State</label>
+		<description>Open window/cold air flow detection</description>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="currentEnergy">
+		<item-type>Number:Power</item-type>
+		<label>Energy Usage</label>
+		<description>Actual energy usage in watts</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%d W"></state>
+	</channel-type>
+	<channel-type id="currentMode">
+		<item-type>String</item-type>
+		<label>Current Room Program Mode</label>
+		<state readOnly="true">
+			<options>
+				<option value="Comfort">Comfort</option>
+				<option value="Sleep">Sleep</option>
+				<option value="Away">Away</option>
+				<option value="Off">Off</option>
+				<option value="AdvancedAway">Vacation away</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="program">
+		<item-type>String</item-type>
+		<label>Program</label>
+		<description>Program associated with room</description>
+		<state readOnly="true" pattern="%s"></state>
+	</channel-type>
+
+	<channel-type id="vacationModeTargetTemperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Target Temperature Vacation</label>
+		<category>Heating</category>
+		<tags>
+			<tag>TargetTemperature</tag>
+		</tags>
+		<state pattern="%d %unit%" min="5" max="35" step="1" />
+	</channel-type>
+	<channel-type id="vacationMode">
+		<item-type>Switch</item-type>
+		<label>Vacation mode</label>
```

Comment:
Label should use uppercase first char word here. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions
```suggestion
		<label>Vacation Mode</label>
```
Commen also applies for labels below.

Reply:
fixed